packages/extension-chakra-store-locator/package.json

```
@@ -41,7 +41,7 @@
   "peerDependencies": {
     "@chakra-ui/react": "^2.8.2",
     "@loadable/component": "^5.15.3",
-    "@salesforce/commerce-sdk-react": "^3.2.0-extensibility-preview.0",
+    "@salesforce/commerce-sdk-react": "^4.0.0-extensibility-preview.0",
```

To match with the version found on line 23 above:
```suggestion
    "@salesforce/commerce-sdk-react": "^4.0.0-extensibility-preview.1",
```